### PR TITLE
Consolodate to one PageHeader component

### PR DESCRIPTION
**Pull request by ZumDeWald**
This PR aims to simplify the headers on each page into one component so all are the same instead of slightly different variations.

Changes:
- Created PageHeader component in the `components/shared` folder
- Changed out header on Quizzes and Contribute page to new PageHeader
- Changed spacing / sizing on header styles to fit for mobile and desktop
- Removed dead components / styles

### CODE REVIEW

**Comment by silentkdev** [comment]
This looks good to me.  

One thing I notice is that there are formatting changes coming through.  Does this mean something is up with the formatter or is someone not using the formatter?

**Comment by ZumDeWald** [comment]
> This looks good to me.
> 
> One thing I notice is that there are formatting changes coming through. Does this mean something is up with the formatter or is someone not using the formatter?

@silentkdev I think the 'Contribute' page Lily did was not formatted properly. We may have still been working out the details of our linting/formatting when it went through

File: client/components/contributepage/InfoSection.js [hunk 1]
```diff
@@ -104,7 +110,10 @@ const InfoSection = () => {
     <Article>
       <BigList>
         <BigItem>
-          <ItemTitle>Choose an {issuesLink}</ItemTitle>
+          <ItemTitle>
+            {/* eslint-disable-next-line */}
```
Comment (Soreg) [inline client/components/contributepage/InfoSection.js:114]: Didn't even know that was a setting - bravo 👍 
We should probably investigate if we should fix the line itself in the future. Normally there's better ways to write it, if eslint complains about it 🙂

File: client/components/quizSelection/styles.js [hunk 2]
```diff
@@ -2,16 +2,6 @@ import styled from "styled-components";
 import { breakpoint } from "../../frontend-config";
 import { PrimaryButton, riseUp } from "../shared/styles";
 
-export const QuizzesHeader = styled.section`
```
Comment (Soreg) [inline client/components/quizSelection/styles.js:5]: Hmm, was this never used? .. I didn't find it anywhere else in the project 🙂

File: client/components/shared/PageHeader.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    @media (min-width: ${breakpoint("md")}) {
+      padding: 40px 0;
+      margin: 0 0 40px 0;
+    }
+  `;
+
+  const PageH1 = styled(Heading1)`
+    color: ${props => props.theme.colors.greenPrimary};
+    padding: 0;
```
Comment (Soreg) [inline client/components/shared/PageHeader.js:25]: Can we add a little padding (ex. 10 px) to the sides? .. We should alwyas have a little space between text, and the edge of the screen (it looks bad if text is lining up against the edge) 🙂

File: client/components/shared/styles.js [hunk 4]
```diff
@@ -13,8 +14,12 @@ export const HugeText = styled.h2`
 export const Heading1 = styled.h1`
   font-style: normal;
   font-weight: bold;
-  font-size: 53px;
-  line-height: 120%;
+  font-size: 35px;
```
Comment (Soreg) [inline client/components/shared/styles.js:17]: Remember that this changes the styling wherever we use this text (also for future references). We just need to be sure that we want to change the general styling of the h1, and not on a particular page 🙂

**Review by Soreg — APPROVED** [review]
Approved (though the padding-comment would be a nice addition)

File: client/components/quizSelection/styles.js (hunk 2 above)
Comment (ZumDeWald) [reply client/components/quizSelection/styles.js:5]: @Soreg this was used on the QuizSelection page when I wrote it, it has been removed in place of the common styling used by the hew shared PageHeader component.

File: client/components/contributepage/InfoSection.js (hunk 1 above)
Comment (ZumDeWald) [reply client/components/contributepage/InfoSection.js:114]: @Soreg I believe Lily got this in before our linting was worked out, so as soon as I opened the file it complained and I could not save it back the way she had it. This was the only way I could get the space to remain between the text and the link.
Like you said, there must be a better way 😅

File: client/components/shared/styles.js (hunk 4 above)
Comment (ZumDeWald) [reply client/components/shared/styles.js:17]: @Soreg I fought with this for a while but when on a mobile device the H1 was just too large. The Figma  designs have it smaller as well (reduced to an H3 there for reference). 
Let me know if you feel it should be larger / handled differently.

File: client/components/shared/PageHeader.js (hunk 3 above)
Comment (ZumDeWald) [reply client/components/shared/PageHeader.js]: Added padding to the H1 here, good suggestion!

**Review by silentkdev — APPROVED** [review]
I thought I had already approved but he's the additional approval!

**Pull request merged**